utilities/dumb_qualisys_joint_center_dictionary_builder.py

```diff
@@ -0,0 +1,37 @@
+
+def dumb_qualisys_joint_center_dictionary_builder() -> dict:
+
+    # this is a special purpose dictionary for getting ARGP skeleton data from qualisys
+    # into a generic meadiapipe++ format
+
+    # below is a handwritten dictionary, where the key represents the joint center name
+    # that is going into the skeleton, the first value represents the physical qualisys
+    # markers that are being combined to be put into the joint center, and the second value
+    # (or array of values) is the relative weight of each of the markers for the joint
+
+    qualisys_joint_center_dict = {
```

Comment:
We should consider switch to making these definitinos in a `yaml` file, which also loads nicely as a `dictionary` but with much friendlier, `markdown`-friendly synctax: 

see: https://www.cloudbees.com/blog/yaml-tutorial-everything-you-need-get-started

Reply:
...or maybe not - There's apparently a whole debate on the 'YAML' vs 'TOML' thing, and it kinda makes me want to stick to 'JSON' for now 😂